### PR TITLE
Create tasks for kickstart presentation

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Manually created as https://github.com/srobo/tasks/issues/398 and https://github.com/srobo/tasks/issues/397.

### CODE REVIEW

File: kickstart/presentation.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        - Software
+        - The docs
+        - The forums
+    6. Health and safety
+        - How to charge your batteries
+    7. Awards
+    8. Schedule for the rest of kickstart
+
+dependencies:
+    - kickstart/rules/write
```
[the remark this answers is not in the document]
Comment (RealOrangeOne) [reply kickstart/presentation.yaml]: To get the *final* presentation, I think it requires the full rules, but in the past I've written much of the rules, and then updated the details of the game closer to the time.

File: kickstart/write-presentation.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    4. General robot design tips
+    5. The kit
+        - Hardware
+        - Software
+        - The docs
+        - The forums
+    6. Health and safety
+        - How to charge your batteries
+    7. Awards
+    8. Schedule for the rest of kickstart
```
Comment (PeterJCLaw) [inline kickstart/write-presentation.yaml]: Unfortunately the formatting of this doesn't survive loading quite as desired. I suggest either using a slightly different style of yaml text block or changing the indent so this is preserved.

You can test to see what the loaded content looks like by running `scripts/import.py kickstart/write-presentation 2020`. currently for this task that shows:
```
#2675: Create kickstart presentation
  Kickstart events require a presentation to go through what student robotics is, what the competition cycle looks like, and details of the game.
  The presentation needs to be given at each venue, so will be presented by multiple people.
  The presentation usually covers:
  1. What student robotics is 2. Who the volunteers are 3. Schedule for the year
      - How long the competition is
      - Tech days date, time, etc..
  4. General robot design tips 5. The kit
      - Hardware
      - Software
      - The docs
      - The forums
  6. Health and safety
      - How to charge your batteries
  7. Awards 8. Schedule for the rest of kickstart

  Original: [recurring-task:kickstart/write-presentation]

  Dependencies:

   * #2674 Write the rules for the SR2020 game
```

**Comment by PeterJCLaw** [comment]
@RealOrangeOne I've just realised that we missed this from this year's import. Would you like to get these fixed and then we can see about adding them to the live tasks?

File: kickstart/write-presentation.yaml (hunk 2 above)
Comment (RealOrangeOne) [reply kickstart/write-presentation.yaml:32]: Fixed with a single character! https://github.com/srobo/recurring-tasks/pull/11/commits/e8326d3c09b0e3612f59d4ea5c5e8e0686789153

Comment (PeterJCLaw) [reply kickstart/write-presentation.yaml:32]: Just to check - have you tried also importing the task into GitHub?

File: kickstart/write-presentation.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+description: |-
+    Kickstart events require a presentation to go through what student robotics
+    is, what the competition cycle looks like, and details of the game.
+
+    The presentation needs to be given at each venue, so will be presented by
+    multiple people.
+
+    The presentation usually covers:
+
+    1. What student robotics is
```
Comment (PeterJCLaw) [inline kickstart/write-presentation.yaml:18]:
```suggestion
    1. What Student Robotics is
```

File: kickstart/decide-presenters.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+milestone: $SRYYYY Kickstart
+
+description: >-
+    The presentation needs to be given at each venue, so will be presented by
+    multiple people. It's recommended the person who creates the presentation be one of the people to
+    present.
+
+dependencies:
+    - kickstart/write-presentation
```
Comment (PeterJCLaw) [inline kickstart/decide-presenters.yaml:15]: Is this strictly a dependency? I realise that https://github.com/srobo/recurring-tasks/issues/25 complicates cross linking, but I think we should probably still aim to keep dependencies as just that.

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**